airbyte-cdk/python/unit_tests/sources/streams/concurrent/test_cursor.py

```diff
@@ -116,7 +116,7 @@ def test_given_boundary_fields_when_close_partition_then_emit_updated_state(self
         self._state_manager.update_state_for_stream.assert_called_once_with(
             _A_STREAM_NAME,
             _A_STREAM_NAMESPACE,
-            {_A_CURSOR_FIELD_KEY: 30},  # State message is updated to the legacy format before being emitted
+            {_A_CURSOR_FIELD_KEY: 0},  # State message is updated to the legacy format before being emitted
```

Comment:
Can we rename the test to explain why the _A_CURSOR_FIELD_KEY is 0? My understanding is that there are no "most_recent_record" for this slice so we use the lower boundary.

As opposition, should we have a test where we close the partition after having observed a record and the _A_CURSOR_FIELD_KEY is the value of the partition?

Reply:
Yes, you are right. We set 0 as the lower boundary and use it as the cursor value. 

Does the `test_given_boundary_fields_and_record_observed_when_close_partition_then_ignore_records` test do what you're asking?

Reply:
It seems so, yes